package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "but-you-promised",
-  "version": "0.0.0",
+  "version": "2.2.0",
```

```suggestion
  "version": "0.0.0",
```
I’d prefer it if this stayed as `0.0.0` as versioning is managed via GitHub releases 🙏

---

Ah, good point 👍 

https://github.com/keirog/but-you-promised/compare/3b803211168a677f688cc13bb37cf6d92134b509..140708aee283b229142f26089dfdfd8512bc279a